### ISSUE TITLE
Wake from deep sleep unreliable

### ISSUE THREAD

**Issue by mattlqx** [mattlqx/gopro-manager#1]
Despite my best efforts, there are occasions where the GoPros just will not wake up after receiving power, this is usually when they're in deep sleep after a day or more of inactivity. The best thing I've come up with so far is just to simply turn on the cameras manually in this scenario.

**Comment by roberto1963** [comment]
Is there batteries inside the gopro's ?
Does this problem also happens when the camera is woken up periodically (for example every hour) ?

**Comment by mattlqx** [comment]
Yes, when battery powered.   If they've been on recently, it isn't a problem. It's the very deep sleep that's a problem. Maybe it has to do with a depleted battery needing a certain charge before the BT circuitry is woken up.
